test/scripts/ephe-domains-delete.sh

```diff
@@ -1,24 +1,12 @@
 #!/bin/bash
```

Comment:
Add `set -eo pipefail` to all shell scripts

```suggestion
#!/bin/bash
set -eo pipefail
```

Reply:
:+1:  thanks, I have added the change to the `common.inc` file.

Reply:
I have replicated this at the beginning of every script.